contracts/schemes/TransitionScheme.sol

```diff
@@ -0,0 +1,88 @@
+pragma solidity 0.5.17;
+
+import "../controller/Controller.sol";
+
+/**
+ * @title A scheme for transitioning the DAO's assets to a new one.
+ */
+
+contract TransitionScheme {
+
+    Avatar public avatar;
+    address payable public newAvatar;
+    address[] public externalTokens;
+    address[] public assetAddresses;
+    bytes4[] public selectors;
+
+    /**
+     * @dev initialize
+     * @param _avatar the avatar to migrate from
+     * @param _newAvatar the avatar to migrate to
+     * @param _externalTokens external tokens to allow transfer to the new avatar
+     * @param _assetAddresses the assets to transfer
+     * @param _selectors the functions to call to to transfer the assets
```

Comment:
Would be good to document here the form a function must take, like what parameters and return value.

Reply:
It is a standard selector per the ABI structure.

Reply:
A selector is an ABI specification for any arbitrary function.  Don't the selectors in this list have to specify a function that has a specific set of parameters and parameter types?